priv/templates/phx.gen.embedded/embedded_schema.ex

```diff
@@ -7,7 +7,9 @@ defmodule <%= inspect schema.module %> do
 <%= for {k, v} <- schema.types do %>    field <%= inspect k %>, <%= inspect v %><%= schema.defaults[k] %>
 <% end %>  end
 
-  @doc false
+  @doc """
+  Builds a changeset based on the `struct` and `params`.
```

Comment:
I think this should be ```Builds a changeset based on the `<%= schema.singular %>` and `attrs`.```

Neither `struct` nor `params` are passed as arguments to the `changeset/2` function :)